### ISSUE TITLE
Can't scroll in the options menu

### ISSUE THREAD

**Issue by verbes4** [ReMinecraftPE/mcpe#107]
Just built for Linux, using scroll wheel, arrow keys, or page up/down keys, I can't seem to scroll down to the "Controls" section of the options screen. 
![Screenshot_20231226_215539](https://github.com/ReMinecraftPE/mcpe/assets/24574201/7fc069f1-ee76-48a6-9d3a-4f97f4886d09)

**Comment by iProgramMC** [comment]
Nope, you have to drag the screen, as if you were using a mobile device. You can add this yourself, or wait until I add it.

This is not a bug, it is a missing feature.

**Comment by verbes4** [comment]
ohh, didn’t realise the feature literally doesnt exist, my bad lol. my main pc, doesn't have a touch screen lol so i had no idea, thank you :)

**Comment by iProgramMC** [comment]
Keeping this issue open because I will fix it later.
